tasks/finalize-buildpack/task.yml

```diff
@@ -7,14 +7,14 @@ image_resource:
 inputs:
   - name: buildpacks-ci
   - name: buildpack
-  - name: pivotal-buildpacks-stack0
-  - name: pivotal-buildpacks-stack1
+  - name: buildpacks.cloudfoundry.org-stack0
```

Comment:
It is not clear to me if the `pivotal-buildpacks-stackX` naming is important or not, and therefore I can't tell if this will break the `finalize-buildpack` task. Given there's no easy way to test this, let's just keep an eye out for this task breaking when it is invoked.